pdfannots/printer/json.py

```diff
@@ -35,6 +35,12 @@ def annot_to_dict(
     if annot.created:
         result['created'] = annot.created.strftime('%Y-%m-%dT%H:%M:%S')
 
+    if annot.color:
+        try:
```

Comment:
Can we delete this try/except block, or do you need it? I can't see any reason for it to fail, and hiding exceptions is a code smell IMO.